tests/data/test_10X_indexes

```diff
@@ -1,2 +1,3 @@
-SI-GA-A1,GGTTTACT,CTAAACGG,TCGGCGTC,AACCGTAA
+SI-GA-A1,CGCGCAG,CTAAACGG,TCGGCGTC,AACCGTAA
 SI-GA-A2,TTTCATGA,ACGTCCCT,CGCATGTG,GAAGGAAC
+SI-NA-A1,TTTCATGA,ACGTCCCT,CGCATGTG,GAAGGAAC
```

Comment:
The same as `SI-NA-A1`. Its sequences should be AAACGGCG,CCTACCAT,GGCGTTTC,TTGTAAGA

Reply:
fixed! 👍